import/popups/WebrtcPermissionDialog.qml

```diff
@@ -0,0 +1,174 @@
+    property alias rememberValue: webrtc.rememberValue
+    property alias choices: webrtc.choices
+
+    property alias acceptText: webrtc.acceptText
+    property alias cancelText: webrtc.cancelText
+
+    WebrtcPermissionInterface {
+        id: webrtc
+
+        anchors.fill: parent
+        rememberValue: remember.checked
+
+        //: Allow the server to use media devices
+        //% "Allow"
+        acceptText: qsTrId("sailfish_components_webview_popups-he-allow_media_devices")
+
+        //: Deny the server access to media devices
+        //% "Deny"
+        cancelText: qsTrId("sailfish_components_webview_popups-he-deny_media_devices")
+
+        readonly property var dictionary: {
+            //% "Front camera"
+            "secondary": qsTrId("sailfish_components_webview_popups-la-primary_camera"),
+            //% "Rear camera"
+            "primary": qsTrId("sailfish_components_webview_popups-la-secondary_camera"),
+            //% "Integrated microphone"
+            "Integrated microphone": qsTrId("sailfish_components_webview_popups-la-integrated_microphone")
+        }
+
+        function translate(word) {
+            var translated = dictionary[word]
+            return translated ? translated : word
+        }
+
+        UserPromptUi {
+            anchors.fill: parent
+            dialog: dialog
+            popupInterface: webrtc
+
+            Column {
+                width: parent.width
+                spacing: Theme.paddingMedium
+
+                Row {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    Icon {
+                        visible: "camera" in devices
+                        source: "image://theme/icon-m-browser-camera"
+                    }
+                    Icon {
+                        visible: "microphone" in devices
+                        source: "image://theme/icon-m-browser-microphone"
+                    }
+                }
+
+                Label {
+                    id: title
+                    wrapMode: Text.WordWrap
+                    x: Theme.horizontalPageMargin
+                    width: parent.width - x * 2
+                    horizontalAlignment: Text.AlignHCenter
+                    font.family: Theme.fontFamilyHeading
+                    font.pixelSize: Theme.fontSizeHuge
+                    color: Theme.highlightColor
+                }
+
+                Label {
+                    id: label
+                    wrapMode: Text.WordWrap
+                    x: Theme.horizontalPageMargin
+                    width: parent.width - x * 2
+                    horizontalAlignment: Text.AlignHCenter
+                    font.pixelSize: Theme.fontSizeSmall
+                    color: Theme.highlightColor
+                }
+
+                ComboBox {
+                    visible: "camera" in devices
+                    x: Theme.horizontalPageMargin
+                    width: parent.width - x * 2
```

Comment:
ComboBoxes have an implicit margin and default to the parent width, so these can be skipped.
```suggestion
```

Reply:
Yes. I missed that small detail. Fixed.